protocol/wasmbinding/msg_plugin.go

```diff
@@ -140,3 +106,33 @@ func parseAddress(addr string) (sdk.AccAddress, error) {
 	}
 	return parsed, nil
 }
+
+func (m *CustomMessenger) placeOrder(
+	ctx sdk.Context,
+	contractAddr sdk.AccAddress,
+	placeOrder *clobtypes.MsgPlaceOrder,
+) ([]sdk.Event, [][]byte, error) {
+	if placeOrder == nil {
+		return nil, nil, wasmvmtypes.InvalidRequest{Err: "Invalid place order request: No order data provided"}
+	}
+	if ctx.IsCheckTx() || ctx.IsReCheckTx() {
```

Comment:
Instead here, we should just panic if CheckTx at the top of `DispatchMsg`? Assuming it's true that Wasm execution doesn't do anything during CheckTx